<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search_CustomDockWidgetBase</class>
 <widget class="QDockWidget" name="Search_CustomDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>968</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NGSC Search</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Search Fields</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="3">
     <widget class="QComboBox" name="searchFieldComboBox"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Comparison</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="layerListComboBox"/>
    </item>
    <item row="13" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Abort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="doneButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="16" column="0" colspan="3">
     <widget class="QTableWidget" name="resultsTable"/>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="suggestionsChecked">
      <property name="text">
       <string>Click for search string autocomplete</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Choose search type or layer</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Predefined or current layer search?</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLabel" name="searchString">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Note:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="chooseType"/>
    </item>
    <item row="7" column="0" colspan="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Search String</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="findStringEdit">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="autocompleteComboBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="12" column="0" colspan="3">
     <widget class="QComboBox" name="comparisonComboBox"/>
    </item>
    <item row="14" column="0" colspan="3">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Previous search results</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1" colspan="2">
     <widget class="QLabel" name="resultsLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
